src/tasks/network/killProcessUsingPort.js

```diff
@@ -0,0 +1,58 @@
+const { ask } = require('KegQuestions')
+const { getProcessesUsingPort, printProcesses, kill } = require('KegUtils')
+const { generalError } = require('KegUtils/error')
+
+/**
+ * Asks user to confirm that they want the processes killed
+ * @param {string} port
+ * @param {Array<Object>} processes - processes to confirm to kill
+ * @returns {Promise<bool>} - user response (true if you should kill the processes)
+ */
+const confirmKill = async (port, processes=[]) => {
+  printProcesses(`Processes using port ${port}:`, processes)
+  return await ask.confirm(`Kill ${processes.length} processes? [y/n]`)
+}
+
+/**
+ * Utility for finding and terminating processes using a specified port
+ * @param {Object} args - arguments passed from the runTask method
+ * @param {string} args.command - Initial command being run
+ * @param {Array} args.options - arguments passed from the command line
+ * @param {Object} args.tasks - All registered tasks of the CLI
+ * @param {Object} globalConfig - Global config object for the keg-cli
+ *
+ * @returns {void}
+ */
+const killProcessUsingPort = async args => {
+  const { params } = args
+  const { port, force } = params
+
+  const processes = await getProcessesUsingPort(port);
+  (processes.length === 0) && generalError(`No processes are using port ${port}`)
```

Comment:
Nothing wrong here, but I would move the `;` to the next line, like this
```js
const processes = await getProcessesUsingPort(port)
;(processes.length === 0) && generalError(`No processes are using port ${port}`)
```

The reasoning for this goes like this:
The first line `const processes = await getProcessesUsingPort(port)` works 100% on its own with out the `;` or any other modification. So it should **NOT** be changed.

The next line `(processes.length === 0) && generalError(`No processes are using port ${port}`)` is what requires the `;` to execute properly. So that's where it should go.

Let me know your thoughts on it?

Reply:
yeah I was frustrated by that semicolon, but I think what you're saying makes more sense. Updated

Reply:
Lol... I've been there. But the more you work **with-out** semicolons, you'll start to learn the places where it's needed.